src/run_analysis.py

```
@@ -99,6 +107,98 @@ def display_config(self) -> None:
         
         print("="*50)
     
+    def _build_query(self, session: Session) -> Any:
```

Pylint (3.3.7)

[refactor] Too many branches (14/12)

(R0912)